kubernetes/generic/bookie.yaml

```diff
@@ -57,7 +57,7 @@ spec:
         spec:
             containers:
               - name: bookie
-                image: streamlio/pulsar:latest
+                image: apachepulsar/pulsar:latest
```

Comment:
I can see both `latest/pulsar:1.19.0-incubating` present at [hub](https://hub.docker.com/r/apachepulsar/pulsar/tags/) so, if we always push latest then it would be better to keep `latest` as it won't require code change.?